srcpkgs/lms/files/lms/run

```diff
@@ -0,0 +1,10 @@
+#!/bin/sh
+[ -r conf ] && . ./conf
+
+: ${USER:=_lms}
```

Comment:
Not exactly sure, but `USER` is usually already set, should probably use some different variable name or reset it before sourcing the conf file. Lets use `LMSUSER` to match other packages.